auxx/Main.hs

```diff
@@ -69,30 +81,47 @@ runNodeWithSinglePlugin ::
 runNodeWithSinglePlugin nr (plugin, plOuts) =
     runNode nr ([plugin], plOuts)
 
-action :: HasCompileInfo => AuxxOptions -> Either (WithCommandAction AuxxMode) Text -> Production ()
-action opts@AuxxOptions {..} command = withConfigurations conf $ do
+action :: HasCompileInfo => AuxxOptions -> Either WithCommandAction Text -> Production ()
+action opts@AuxxOptions {..} command = do
     CLI.printFlags
-    logInfo $ sformat ("System start time is "%shown) $ gdStartTime genesisData
-    t <- currentTime
-    logInfo $ sformat ("Current time is "%shown) (Timestamp t)
-    (nodeParams, tempDbUsed) <-
-        correctNodeParams opts =<< CLI.getNodeParams cArgs nArgs
     let
-        toRealMode :: AuxxMode a -> RealMode EmptyMempoolExt a
-        toRealMode auxxAction = do
-            realModeContext <- ask
-            let auxxContext =
-                    AuxxContext
-                    { acRealModeContext = realModeContext
-                    , acCmdCtx = cmdCtx
-                    , acTempDbUsed = tempDbUsed }
-            lift $ runReaderT auxxAction auxxContext
-    let vssSK = unsafeFromJust $ npUserSecret nodeParams ^. usVss
-    let sscParams = CLI.gtSscParams cArgs vssSK (npBehaviorConfig nodeParams)
-    bracketNodeResources nodeParams sscParams txpGlobalSettings initNodeDBs $ \nr ->
-        runRealBasedMode toRealMode realModeToAuxx nr $
-            (if aoNodeEnabled then runNodeWithSinglePlugin nr else identity)
-            (auxxPlugin opts command)
+        runWithoutNode = do
+          rawExec Nothing opts Nothing command
+
+    let configToDict :: HasConfigurations => Production (Maybe (Dict HasConfigurations))
+        configToDict = return (Just Dict)
+
+    hasConfigurations <- case aoStartMode of
+        Automatic -> (flip catch) (\(_ :: ConfigurationException) -> return Nothing)
+                    . (flip catch) (\(_ :: ConfigurationError)     -> return Nothing)
```

Comment:
Also I think it'd be good to print a message like:
> Using light mode because \<reason\>

because sometimes user may be sure they have valid configuration, but something may be unexpectedly wrong.

Reply:
I thought about that, but that makes it more complicated to use in scripts, where sometimes we want to check the output of a specific command. Not sure if that's even an issue or not, so I'll add the message.

Reply:
So now you print the mode, but don't print the reason. I think printing the reason is useful, because you may be sure that everything is ok for `with-config` mode, but for some reason `Light` one is used.
But I also realized that then you can pass `--mode with-config` and you will see the error message, so ok.